dash-renderer/tslint.json

```diff
@@ -0,0 +1,58 @@
+{
+    "defaultSeverity": "error",
+    "extends": [
+        "tslint:recommended"
+    ],
+    "linterOptions": {
+        "exclude": [
+            ".config/**",
+            "cypress/**",
+            "inst/**",
+            "node_modules/**",
+            "@Types/**",
+            "venv/**",
+            "**/*.js"
+        ]
+    },
+    "rules": {
+        "array-type": false,
+        "arrow-parens": [true, "ban-single-arg-parens"],
+        "ban-types": false, // to allow use of 'function' type
```

Comment:
Can you actually use comments here (and below)? Normally not allowed by JSON.

Reply:
In VSCode I have this file handled as a `JSON with Comments`, removing the comments https://github.com/plotly/dash/pull/1254/commits/a5f9c3148c6c05342dff1054bed2c320c12403dc